cmd/gaia/cli_test/test_helpers.go

```diff
@@ -163,7 +163,7 @@ func (f *Fixtures) Flags() string {
 
 // UnsafeResetAll is gaiad unsafe-reset-all
 func (f *Fixtures) UnsafeResetAll(flags ...string) {
-	cmd := fmt.Sprintf("gaiad --home=%s unsafe-reset-all", f.GDHome)
+	cmd := fmt.Sprintf("../../../build/gaiad --home=%s unsafe-reset-all", f.GDHome)
```

Comment:
We should add an `init()` check to set the binary name for the tests. CI sets some ENV so using those would fix any failures:

```go
var (
	gdbin   = "gaiad"
	gclibin = "gaiacli"
)

func init() {
	if ci, _ := strconv.ParseBool(os.Getenv("CI")); ci {
		gdbin = "../../../build/gaiad"
		gclibin = "../../../build/gaiacli"
	}
}
```

Or we could also do this check in the `NewFixtures` function. I'm happy to push a commit tomorrow to do this.